snapm/command.py

```
@@ -716,6 +717,89 @@ def prune_snapset(manager, name, sources):
+                      system) to use as the right side of the comparison.
+    :type diff_to: ``str``
+    :param options: Options controlling the comparison.
+    :type options: ``DiffOptions``
+    :returns: A list of file system diff records.
+    :rtype: ``List[FsDiffRecord]``
+    """
+    if diff_from == diff_to:
+        target = f"'{diff_from}'" if diff_from != "." else "system root"
+        raise ValueError(
+            f"Cannot compare {target} to itself.",
+        )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)